### ISSUE TITLE
Fix TEPs link construction in cross-references section and remove summary widget and detail view

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#1753]
Currently, we generate a link to the SGC if the target has a TEP. The link appears in the cross-references section of the target profile page.

![Screenshot 2021-09-24 at 16 24 11](https://user-images.githubusercontent.com/7490258/134700526-9c53aae3-764e-4f2b-8704-d797407971ab.png)

These links are generated by the [TepLink.js](https://github.com/opentargets/platform-app/blob/main/src/components/ExternalLink/TepLink.js) file and use an object of gene IDs, symbols, and links to construct the link — see [lines 19 - 160](https://github.com/opentargets/platform-app/blob/e2d9f4d431cd9c1916de835d5e985439e576c8ab/src/components/ExternalLink/TepLink.js#L19).

Can we please update the link construction logic to use the GraphQL API? 

TEP information is available from the `target` endpoint:

```
query tepsInfoForRIPK2 {
  target(ensemblId: "ENSG00000104312") {
    id
    approvedSymbol
    tep{
      name
      uri
      therapeuticArea
      description
    }
  }
}

```
Query: [RIPK2](https://api-beta-dot-open-targets-eu-dev.ew.r.appspot.com/api/v4/graphql/browser?query=query%20tepsInfoForRIPK2%20%7B%0A%20%20target%28ensemblId%3A%20%22ENSG00000104312%22%29%20%7B%0A%20%20%20%20id%0A%20%20%20%20approvedSymbol%0A%20%20%20%20tep%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20uri%0A%20%20%20%20%20%20therapeuticArea%0A%20%20%20%20%20%20description%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

Once we have adjusted the TepLink.js file to use the GraphQL API, can we also please remove the TEP summary widget and detail view?

**Comment by andrewhercules** [comment]
@carcruz can you please work on this and adjust how we construct TEP links?

**Comment by andrewhercules** [comment]
Ticket closed as work completed, merged into `main`, and included in the [platform-app `21.09.5` release](https://github.com/opentargets/platform-app/releases/tag/21.09.5)
